### PR TITLE
Display max TPS from all nodes at end of client demo

### PR DESCRIPTION
**Pull request by pgarg66**
- Also lists node with 0 TPS and overall average TPS

### CODE REVIEW

File: src/bin/client-demo.rs [hunk 1]
```diff
@@ -335,12 +338,30 @@ fn main() {
     // Compute/report stats
     let mut max_of_maxes = 0.0;
     let mut total_txs = 0;
-    for (max, txs) in maxes.read().unwrap().iter() {
+    let mut nodes_with_zero_tps = 0;
+    let mut total_maxes = 0.0;
+    for (max, txs, sock) in maxes.read().unwrap().iter() {
+        println!("Node:{}, Max TPS: {:.2}", *sock, *max);
+        if *max == 0.0 {
```
Comment (sakridge) [inline src/bin/client-demo.rs]: Use `txs == 0` here? exact compare with float makes me cringe, although probably ok here.

Comment (pgarg66) [reply src/bin/client-demo.rs]: I'll change it to txs == 0

File: src/bin/client-demo.rs [hunk 2]
```diff
@@ -33,9 +33,14 @@ use std::thread::JoinHandle;
 use std::time::Duration;
 use std::time::Instant;
 
+pub struct NodeStats {
+    pub tps: f64,
+    pub tx: u64,
```
Comment (mvines) [inline src/bin/client-demo.rs]: can you please document these two fields?  I can mostly guess what they mean but I'd rather not guess :)

Comment (pgarg66) [reply src/bin/client-demo.rs]: Done..

**Pull request merged**